apps/festival/festival/src/app/dashboard/title/list/list.component.html

```diff
@@ -31,22 +31,49 @@ <h1>My Titles</h1>
       </button>
     </header>
 
-    <bf-table [source]="result$ | async" pagination="50" useFilter  clickable (rowClick)="goToTitle($event)">
-      <ng-template colRef="title.international" label="Title" let-title sort>
-        <b>{{ title }}</b>
-      </ng-template>
-      <ng-template colRef="analytics" label="# Views" let-analytics sort>
-        {{ analytics?.length }}
+    <bf-table [source]="result$ | async" pagination="50" useFilter>
+      <ng-template colRef label="Title" let-movie sort>
+        <a [routerLink]="movie.id" [matTooltip]="movie.title.international">
+          <u>{{ movie.title.international | maxLength:50 }}</u>
+        </a>
       </ng-template>
-      <ng-template colRef="directors" label="Director(s)" let-directors>
-        {{ directors | displayName | maxLength:50 }}
+      <ng-template colRef label="Director(s)" let-movie>
+        {{ movie.directors | displayName | maxLength:50 }}
       </ng-template>
-      <ng-template colRef="productionStatus" label="Production Status" let-prod>
+      <ng-template colRef="productionStatus" label="Production Status" let-prod sort>
         {{ prod | toLabel: 'productionStatus' }}
       </ng-template>
       <ng-template colRef="app.festival.status" label="Status" let-status>
         <bf-tag>{{ status | toLabel: 'storeStatus' }}</bf-tag>
       </ng-template>
+
+      <ng-template *ngIf="(filter$ | async) === 'archived'" colRef label="Actions" let-movie sticky>
+        <div class="action-buttons">
+          <a [routerLink]="movie.id" mat-icon-button matTooltip="View">
+            <mat-icon svgIcon="visibility"></mat-icon>
+          </a>
+          <a [routerLink]="['/c/o/dashboard/tunnel/movie', movie.id]" mat-icon-button matTooltip="Edit">
+            <mat-icon svgIcon="edit"></mat-icon>
+          </a>
+          <button type="button" mat-icon-button (click)="changeMovieStatus(movie, 'draft')" matTooltip="Restore">
+            <mat-icon svgIcon="restore"></mat-icon>
+          </button>
+        </div>
+      </ng-template>
+
+      <ng-template *ngIf="(filter$ | async) !== 'archived'" colRef label="Actions" let-movie sticky>
+        <div class="action-buttons">
+          <a [routerLink]="movie.id" mat-icon-button matTooltip="View">
+            <mat-icon svgIcon="visibility"></mat-icon>
+          </a>
+          <a [routerLink]="['/c/o/dashboard/tunnel/movie', movie.id]" mat-icon-button matTooltip="Edit">
+            <mat-icon svgIcon="edit"></mat-icon>
+          </a>
+          <button type="button" mat-icon-button (click)="changeMovieStatus(movie, 'archived')" matTooltip="Archive">
+            <mat-icon svgIcon="archive"></mat-icon>
+          </button>
+        </div>
+      </ng-template>
```

Comment:
Not possible to do something like this ?
```suggestion
      <ng-template  colRef label="Actions" let-movie sticky>
        <div class="action-buttons">
          <a [routerLink]="movie.id" mat-icon-button matTooltip="View">
            <mat-icon svgIcon="visibility"></mat-icon>
          </a>
          <a [routerLink]="['/c/o/dashboard/tunnel/movie', movie.id]" mat-icon-button matTooltip="Edit">
            <mat-icon svgIcon="edit"></mat-icon>
          </a>
          <button  *ngIf="movie.status === 'archived' else archive type="button" mat-icon-button (click)="changeMovieStatus(movie, 'draft')" matTooltip="Restore">
            <mat-icon svgIcon="restore"></mat-icon>
          </button>
          <ng-template #archive>
            <button type="button" mat-icon-button (click)="changeMovieStatus(movie, 'archived')" matTooltip="Archive">
            <mat-icon svgIcon="archive"></mat-icon>
          </button>
          </ng-template>
        </div>
      </ng-template>
```

Reply:
I will try, that the version of Remco

Reply:
Not working